w09h01/test/pgdp/pingutrip/TransformToWaysTest.java

```diff
@@ -0,0 +1,24 @@
+package pgdp.pingutrip;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class TransformToWaysTest {
+    @Test
+    void testTransformToWays() {
+        List<WayPoint> points = List.of(new WayPoint(4.0, 11.5), new WayPoint(19.1, 3.2));
+        List<OneWay> ways = PinguTrip.transformToWays(points).toList();
+
+        OneWay[] expected = { new OneWay(points.get(0), points.get(1)) };
+        OneWay[] actual = ways.toArray(OneWay[]::new);
+
+        assertEquals(expected.length, actual.length,
+                "The stream should contain exactly " + expected.length + " values");
+        for (int i = 0; i < expected.length; i++) {
```

Comment:
Couldn't you just use assertArrayEquals here?

Reply:
Oh, of course `assertArrayEquals` is better 👍 

I have also updated the values because the example in Artemis has been changed.